src/Targets/Microsoft.Managed.DesignTime.targets

```diff
@@ -54,7 +55,38 @@
     <ProjectCapability Include="LanguageService" />
     <ProjectCapability Include="ProjectConfigurationsInferredFromUsage" />
  </ItemGroup>
-  
+
+  <!-- Common Project System rules that override rules defined in CPS. These are exact copy of the rules defined in CPS. -->
+  <ItemGroup>
```

Comment:
We should define a DefineCommonManagedItemSchemas and make the Itemgroup conditional on that property and include items from the below itemgroup that are replacing the Common one. We'll likely need to set DefineCommonItemSchemas=false for shared projects. Can you test shared projects with this change.